code/game/machinery/iv_drip.dm

```diff
@@ -132,6 +132,8 @@
 				update_icon()
 
 /obj/machinery/iv_drip/attack_hand(mob/user as mob)

```

Comment:
As someone who played a medborg, being able to remove the bloodbag just gave me false hope that I could then add one. It was rather fruitless.